cmake/InstallLayout.cmake

```diff
@@ -284,10 +284,11 @@ set(deb-server-filename "foundationdb-server_${PROJECT_VERSION}.${CURRENT_GIT_VE
 set(CPACK_RPM_PACKAGE_LICENSE "Apache 2.0")
 
 set(CPACK_RPM_PACKAGE_NAME "foundationdb")
-set(CPACK_RPM_CLIENTS-EL6_PACKAGE_NAME "foundationdb-clients")
-set(CPACK_RPM_CLIENTS-EL7_PACKAGE_NAME "foundationdb-clients")
-set(CPACK_RPM_SERVER-EL6_PACKAGE_NAME "foundationdb-server")
-set(CPACK_RPM_SERVER-EL7_PACKAGE_NAME "foundationdb-server")
+set(CPACK_RPM_CLIENTS-EL6_PACKAGE_NAME "foundationdb-clients-el6")
```

Comment:
I don't think this is a good idea - and it will break our current setup (we use RPMs internally). This will rename the packages to `foundation-clients-el6` which has several implications:

1. Existing `foundation-clients` will be recognized as a different package. Therefore, if it is already installed, it won't upgrade and instead it will result in two installations of the same package (maybe it will conflict on some files?).
1. Packages that depend on `foundationdb-clients` or `foundationdb-server` will break as these won't now exist anymore in the new versions.
1. Upgrading from EL-6 to EL-7 will be very awkward in general...

Reply:
I agree. I was hoping to make changes that would allow the creation of RPMs for el6 and el7 from a single command. Turns out that I was copying the package as its original name but the new name was being preserved within the rpm.